.github/workflows/release.yaml

```diff
@@ -36,9 +57,11 @@ jobs:
       id: meta
       uses: docker/metadata-action@v4
       with:
-        images: ghcr.io/akuity/kargo
+        images: ${{ steps.repo.outputs.repo }}
         flavor: latest=false
-        tags: type=semver,pattern={{raw}}
+        tags: |
+          type=semver,pattern={{raw}}
```

Comment:
Should this line be applicable only when the event is a release?

Reply:
It is not required, a scheduled [event](https://github.com/docker/metadata-action#priority-attribute) has a higher priority attribute which will cause this tag to be skipped.  I have tested the entire workflow in a private repo for testing.  I have confirmed this is working as expected.   We could be explicit and add a check if you want.